### PR TITLE
Added command to get/set packet size for gdb io system

### PR DESCRIPTION
**Pull request by SrimantaBarua**

### CODE REVIEW

File: libr/io/p/io_gdb.c [hunk 1]
```diff
@@ -196,10 +197,30 @@ static int __system(RIO *io, RIODesc *fd, const char *cmd) {
...
-                         " =!exec_file [pid] - Get file which was executed for"
+                         " =!pktsz           - get max packet size used\n"
+                         " =!pktsz bytes     - set max. packet size as 'bytes' bytes\n"
+                         " =!exec_file [pid] - get file which was executed for"
                                              " current/specified pid\n");
 		return true;
 	}
+	if (!strncmp (cmd, "pktsz", 5)) {
+		char *ptr = cmd + 5;
+		while (isspace (*ptr)) {
```
Comment (radare) [inline libr/io/p/io_gdb.c]: use r_str_* to skip spaces (trim/chop_ro)

Comment (radare) [inline libr/io/p/io_gdb.c]: to skip tabs and other spacing chars

File: libr/io/p/io_gdb.c [hunk 2]
```diff
@@ -196,10 +197,30 @@ static int __system(RIO *io, RIODesc *fd, const char *cmd) {
...
                          " =!inv.reg         - invalidate reg cache\n"
-                         " =!exec_file [pid] - Get file which was executed for"
+                         " =!pktsz           - get max packet size used\n"
+                         " =!pktsz bytes     - set max. packet size as 'bytes' bytes\n"
+                         " =!exec_file [pid] - get file which was executed for"
                                              " current/specified pid\n");
 		return true;
 	}
+	if (!strncmp (cmd, "pktsz", 5)) {
+		char *ptr = cmd + 5;
```
Comment (radare) [inline libr/io/p/io_gdb.c]: this is const char *

**Comment by XVilka** [comment]
Is there any way to run it before connecting? Why you doesn't want to use config variable?

**Comment by radare** [comment]
maybe with an env var?

**Comment by SrimantaBarua** [comment]
@XVilka the config var callback idea would be messy because it's an IO plugin. So either we have a check for gdb inside r_io (untidy), or we would have to set maybe a field in the `RIO` struct, and then the gdb plugin would have to poll that every time. And code in `debug_gdb` would have to check too maybe with iobind.

**Pull request merged**